### ISSUE TITLE
Image creation error

### ISSUE THREAD

**Issue by jdaezone** [danilodelucio/NukeDiffusion#6]
![캡처](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/0ee8288f-e52d-4ae9-b843-d29b167700c5)

Hello, I'm a VFX compositor.
Thank you so much for providing such good information.

I've finished installing the plug-in, and I'm trying to use the inpaint function, but as a result, the image has not been created.

May I ask you about the solution?

**Comment by danilodelucio** [comment]
Hi @jdaezone , thanks for using the tool, I appreciate your feedback!

It's difficult to figure out the issue, could you please send some screenshots from the NukeDiffusion Terminal? It would be useful to check the information.

Just to make sure, is it working for the other workflows? (**img2img**/**txt2img**)

**Comment by jdaezone** [comment]
![1](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/77696b01-4e24-4517-8a74-9e5cec8c4440)

The image above is an inpainting test image, but when I tested it again, there was an error related to the mask.



![2](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/47af5c24-ff01-4da8-9339-9bdf99eb2de4)
![3](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/eb68b725-de2e-4a1a-bbb9-2f4f04deea35)

The above image is a test image related to txt2img.



![4](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/f7766ff8-dbd4-44f4-9d02-92d87b20a945)
![5](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/ed50e4b5-bbb1-4e16-bf3b-9c936f3593e4)

The above image is a test image related to img2img.



![6](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/9096038b-fa6b-460b-b57d-cf407c909e2e)

In conclusion, the image is created like this, so what is the solution?

**Comment by danilodelucio** [comment]
Thanks for providing the screenshots!

Here are some questions...

1- When you opened this Issue, you mentioned that it was not working for the Inpainting workflow only. However, from these provided screenshots, it seems like the other workflows are not working either. Is that correct?;

2- Could you check the **_input** folder and see if Nuke rendered your input images? This is because the Inpainting workflow will render your inputs from the NukeDiffusion node (this is just to check if it is rendering them or not);

3- Could you try a different Checkpoint? Maybe using an SD model instead of SDXL;

4- Lastly, just to make sure, did you wait for the whole setup installation process until you see the "The NukeDiffusion setup has been completed!"? I know this looks obvious, but some people didn't wait for it, and some dependencies were not installed properly;

**Comment by jdaezone** [comment]
1,3. Yes, it doesn't work in all workflows (txt2img, img2img, Inpainting, SDXL/SD)
2. No additional images have been rendered in the input folder.
4. I re-installed Pytorch and Nuke Diffusion. Is it related to the image not being created? Please check the image below..

![8](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/cd04f4ad-addb-43ab-bcde-5928b6ec02a8)
![9](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/ed77865a-5a91-4320-b960-46814559c219)
*Pytorch

![10](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/2a4abc9c-900a-43e2-b7f2-7b8c02c7937e)
*install_or_update_dependencies

**Comment by danilodelucio** [comment]
Thanks for providing the answers!

Well, honestly, it's a bit weird because from your screenshots everything seems to be correct.

The only thing I could say is that I think you are using the older version of NukeDiffusion. I uploaded the v01.1 last month, and looking at your screenshots I see a different folder structure.

In the updated version, we don't have the **cuda** folder and the **Python** built-in anymore. I highly recommend that you download it and install it again (https://github.com/danilodelucio/NukeDiffusion).

If you do that, make sure to have a Python version 3.8-3.11 installed into your system. If you need help with any of these steps don't hesitate to ask!

I'm sorry for taking your time doing all this process, I'm trying my best. 🙁

**Comment by jdaezone** [comment]
Thank you so much for letting me know about the solution :)

I installed a new version and retested it, but the same error repeats...

Should I proceed with the test on another PC?

**Comment by danilodelucio** [comment]
No worries! Yeah if possible it would be great to test it on another computer, because I don't know why it is not working for you (really sorry for that).

For all the information you provided, it does not look to be the tool I made, maybe it's a conflict with some Stable Diffusion dependencies.

If you installed the new version, could you confirm if the input images were rendered inside the **_input** folder? Also, could you confirm which version of Python you are using? (if you can send screenshots would be better)

**Comment by jdaezone** [comment]
After rendering the image (input), I proceeded to create the image, but the same problem occurred as before.

Python version is using version 3.11.9 as shown in the terminal...!

![캡처](https://github.com/danilodelucio/NukeDiffusion/assets/167772956/3b4a0028-44bf-4e12-a89e-9f7736f65f20)
